### ISSUE TITLE
Exception when formatting array with trailing comma

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#37]
When attempting to format:

```
FUNCTION_BLOCK SomePou EXTENDS FB_TestSuite
VAR
		arrayWithValues : ARRAY[1..20] OF LREAL := [
		328212, 313818, 302989, 304328, 320842, 342858, 324057, 278686, 165224, 109087,
		107209, 103290, 103762, 118398, 120980, 120038, 70042, 33123, 33270, 34880,
	];
END_VAR
```

I get the following System.InvalidOperationException:

![image](https://user-images.githubusercontent.com/20192767/113708479-8f3ed600-96e1-11eb-88fc-782973af0af6.png)

And no formatting is done. If I remove the trailing comma after the last number it properly formats.

**STweep version info**
STweep v3.3.1 for TwinCAT

**IDE version info**
Microsoft Visual Studio Community 2017 Version 15.9.33

**Comment by Barteling** [comment]
Thanks for reporting.
Ill check what's going on

**Comment by Barteling** [comment]
Fixed in STweep for TwinCAT 3.3.2 ->  https://www.stweep.com/releases/
Codesys update will follow soon.

**Comment by Roald87** [comment]
Got a similar exception:

![image](https://user-images.githubusercontent.com/20192767/114671840-d2620000-9d04-11eb-872e-975167302e7e.png)

when attempting to format 

```
FUNCTION_BLOCK POU
VAR
	Parameter : ARRAY[1..3] OF ST_Parameters := [
		(A:=1, B:=1, B:=1),
		(A:=1, B:=1, B:=1), 
		(A:=1, B:=1, B:=1)
	];
END_VAR
```

Using [settings.txt](https://github.com/Barteling/STweep/files/6309047/settings.txt) and STweep v3.4.0
